Steps.md

```diff
@@ -0,0 +1,25 @@
+Git tips
+========
+Overview
+These steps summarize what will be done from the cloning of remote part to your team as well as the realization of changes in both remote and local.
+
+ 1. Git clone
+	`git clone https://github.com/gustavonecore/wiki.wiki.git`
+ 2. Creation new branch
+ `git checkout -b feature/1 master`
+ 3. Change your code here...	
+ 4. Status archives 
+    Lookup modified/added files from your local work
+	`git status`
+ 5. Add archives 
+ Add files to the commit `git add file1 file2 file4` or you can `add all git add .`
```

Comment:
Add files to the commit 
`git add file1 file2 file4...` or you can add all the modified files with `git add .`